db/migrate/20220310235942_create_admins.rb

```diff
@@ -0,0 +1,13 @@
+class CreateAdmins < ActiveRecord::Migration[7.0]
+  def change
+    create_table :admins do |t|
+      t.string :name
+      t.string :email
+      t.string :position
+      t.integer :private_number
+      t.string :password
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
I guess this table `:admins` and `:employees` are going to store somehow unique unrepeatable users, right? I mean you can not have two users with the same email but different name, so if that is true, I think as a good practice you have to add indexes to your tables in order to keep that consistency in the data

Reply:
I agree with @vyk2rr , since if you implement a session management gem such as [Devise](https://github.com/heartcombo/devise) or [Clearance](https://github.com/thoughtbot/clearance), the gem will automatically generate a user table with the email field, which would generate inconsistency with the email data that you have already generated in your tables.

Reply:
Done on the new commit.